we don't need tools when collecting profiles with opt-dist
--- a/src/bootstrap/src/core/build_steps/llvm.rs
+++ b/src/bootstrap/src/core/build_steps/llvm.rs
@@ -349,6 +349,7 @@
         cfg.define("LLVM_INSTALL_UTILS", "ON");
 
         if builder.config.llvm_profile_generate {
+            cfg.define("LLVM_INCLUDE_TOOLS", "OFF");
             cfg.define("LLVM_BUILD_INSTRUMENTED", "IR");
             if let Ok(llvm_profile_dir) = std::env::var("LLVM_PROFILE_DIR") {
                 cfg.define("LLVM_PROFILE_DATA_DIR", llvm_profile_dir);
